cmd/bricksllm/.env

```diff
@@ -0,0 +1,27 @@
+CONFIG_FILE_NAME=./config_local.json
+
+POSTGRESQL_HOSTS=localhost
+POSTGRESQL_DB_NAME=postgres
+POSTGRESQL_USERNAME=postgres
+POSTGRESQL_PASSWORD=postgres
+POSTGRESQL_SSL_MODE=disable
+POSTGRESQL_PORT=5432
+REDIS_HOSTS=localhost
+REDIS_PORT=6379
+REDIS_USERNAME=
+REDIS_PASSWORD=eYVX7EwVmmxKPCDmwMtyKVge8oLd2t81
+REDIS_READ_TIMEOUT=1s
```

Comment:
this value should be REDIS_READ_TIME_OUT.  it was a typo but it cannot be changed due to backward compatibility

Reply:
done.